### PR TITLE
Add strace examples

### PR DESCRIPTION
**Pull request by tyhawkins**
Adds `strace` examples.

### CODE REVIEW

**Comment by srsudar** [comment]
Thanks! I don't know how to use strace myself, so this will be useful.

In the first section we try to use only an example that would work exactly as typed, without the `<arg>` style notation. Do you mind tweaking it with that in mind? Take a look at [find.md](https://github.com/srsudar/eg/blob/master/eg/examples/find.md) for examples. In the `strace` case, eg, rather than:

```
strace -f <command>
```

it would be:

```
strace -f ls
```

(Or whatever the correct command is.)

Then in the `Basic Usage` section we use the `<arg>` format like you have it. I'd offer to edit it myself but I don't have my linux machine handy atm.

**Comment by tyhawkins** [comment]
@srsudar Updated the PR.

**Comment by srsudar** [comment]
Sorry for the delay on this. I haven't forgotten. Will try to get to it soon.

File: eg/examples/strace.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
+# strace
+
+trace syscalls of a command and follow child pids
+
+    strace -f ls
+
+# Basic Usage
```
Comment (scorphus) [inline eg/examples/strace.md:7]: Three lines before main sections:

```suggestion


# Basic Usage
```

File: eg/examples/strace.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+Show time spent in syscalls
+
+    strace -T <command>
+
+Only show a summary of syscalls
+
+    strace --summary-only/-c
+
+# Attach to a running process to see time spent in calls
```
Comment (scorphus) [inline eg/examples/strace.md:41]: Three lines before main sections:

```suggestion


# Attach to a running process to see time spent in calls
```

File: eg/examples/strace.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+    strace -f -s 256 -p <pid> -T
+
```
Comment (scorphus) [inline eg/examples/strace.md:44]: Would be nice to have a brief explanation of the command below. See other examples as reference.

File: eg/examples/strace.md [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+    ~$ strace -f -s 256 -T echo "hi"
```
Comment (scorphus) [inline eg/examples/strace.md:45]:
```suggestion
    $ strace -f -s 256 -T echo "hi"
```

**Review by scorphus — CHANGES_REQUESTED** [review]
Thanks for contributing, @tyhawkins.

I can't yet comment on `strace` usage itself, as I'm on a Mac. But, for now, I can comment on styling. Please consider my sugestions.

File: eg/examples/strace.md (hunk 3 above)
Comment (tyhawkins) [reply eg/examples/strace.md:48]: Added some more information about the command.

**Comment by srsudar** [comment]
Sorry for the delay on this @tyhawkins. Merging now, and I'll send it out in the next release (probably in a few weeks). Sorry for being so slow.

**Pull request merged**